ceph_deploy/mds.py

```diff
@@ -122,6 +122,8 @@ def create_mds(distro, name, cluster, init):
     if distro.is_el:
         system.enable_service(distro.conn)
 
+    if distro.distro == 'suse':
+        system.enable_service(distro.conn, 'ceph')
```

Comment:
@osynge point taken!  That seems like a good suggestion.

There are three instances of is_el in use -- two of them are here (osd.py and mds.py), and the other is rgw.py.  If it makes sense to change it for RGW as well (take a look at it) might as well do all three of them.

Thanks!